### PR TITLE
Himss branch 2020

### PR DESCRIPTION
**Pull request by KeeyanGhoreshi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,10 +4,11 @@
   "private": true,
   "scripts": {
     "build": "webpack --config ./webpack.config.prod.js --progress --colors",
-    "start": "webpack-dev-server --config ./webpack.config.dev.js",
+    "start": "webpack-dev-server --config ./webpack.config.dev.js & json-server --watch ./src/db.json --port 3006",
     "startDev": "webpack-dev-server --config ./webpack.config.dev.js",
     "startProd": "webpack-dev-server --config ./webpack.config.prod.js",
     "startTemplate": "webpack-dev-server --config ./webpack.config.run.js",
+    "startTest": "node ./setup.js",
```
Comment (KeeyanGhoreshi) [inline package.json]: Should probably make this the production start

File: src/components/QuestionnaireForm/QuestionnaireForm.jsx [hunk 2]
```diff
@@ -1058,12 +1061,14 @@ export default class QuestionnaireForm extends Component {
               Next
             </button>
           )} */}
-          <button
+           {/*
```
Comment (KeeyanGhoreshi) [inline src/components/QuestionnaireForm/QuestionnaireForm.jsx]: We can probably add this button back if we're not doing a demo anytime soon

File: setup.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+//     server.use(jsonServer.bodyParser);
+// server.use((req, res, next) => {
+//   if (req.method === "POST") {
+//     req.body.createdAt = Date.now();
+//   }
+//   // Continue to JSON Server router
+//   next();
+// });
+
+//     server.get("/register", function(req, res){
```
Comment (plarocque4) [inline setup.js:45]: Can these be removed since they are down below?

File: setup.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+      }})).pipe(res);
+  });
+  server.use("/getfile", function(req, res) {
+    var url = "https://davinci-crd.logicahealth.org/getfile" + req.url;
+    req.pipe(request({url:url,  agentOptions: {
+        rejectUnauthorized: false
+      }})).pipe(res);
+  });
+
+// pem.createCertificate({ days: 1, selfSigned: true }, function (err, keys) {
```
Comment (plarocque4) [inline setup.js:25]: Is this needed if not used?

File: src/components/QuestionnaireForm/QuestionnaireForm.jsx (hunk 2 above)
Comment (plarocque4) [reply src/components/QuestionnaireForm/QuestionnaireForm.jsx:667]: Do we want the button to be in the final checked in version?

**Comment by plarocque4** [comment]
Can you forward merge the latest master into this?

**Comment by plarocque4** [comment]
Have you tried the latest from all the other repos with this?

**Comment by lpdecelles** [comment]
Yes, this has conflicts. A forward merge might fix it.

**Comment by lpdecelles** [comment]
Looks good. I tested it with HOT R4.

**Pull request merged**